code/__HELPERS/game.dm

```diff
@@ -567,7 +567,7 @@
 	if(!istype(environment))
 		return
 	var/pressure = environment.return_pressure()
-	if(pressure <= LAVALAND_EQUIPMENT_EFFECT_PRESSURE)
+	if(pressure <= LAVALAND_EQUIPMENT_EFFECT_PRESSURE & pressure > 0)
```

Comment:
Use a define for this and make it slightly below the minimum lavaland pressure rather than 0 (1kPa shouldnt count too)